### PR TITLE
[README] Config Github codespace and Dev container

### PR DESCRIPTION
**Pull request by LeonZh0u**

### CODE REVIEW

File: .devcontainer/README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+You may use the button above to open this repo in a Codespace
+
+For more info, check out the [GitHub documentation](https://docs.github.com/en/free-pro-team@latest/github/developing-online-with-codespaces/creating-a-codespace#creating-a-codespace).
+  
+## VS Code Dev Containers
+<a href="https://vscode.dev/redirect?url=vscode://ms-vscode-remote.remote-containers/cloneInVolume?url=https://github.com/geekan/MetaGPT"><img src="https://img.shields.io/static/v1?label=Dev%20Containers&message=Open&color=blue&logo=visualstudiocode" alt="Open in Dev Containers"></a>
+
+Note: If you click this link you will open the main repo and not your local cloned repo, you can use this link and replace with your username and cloned repo name: 
+https://vscode.dev/redirect?url=vscode://ms-vscode-remote.remote-containers/cloneInVolume?url=https://github.com/<yourusername>/<yourclonedreponame>
```
Comment (voidking) [inline .devcontainer/README.md:18]: This link should be surrounded by accents. Or `<yourusername>/<yourclonedreponame>` will disappear after rendering.

File: .devcontainer/README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+You can use the dev container configuration in this folder to build and run the app without needing to install any of its tools locally! You can use it in [GitHub Codespaces](https://github.com/features/codespaces) or the [VS Code Dev Containers extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers).
+
+## GitHub Codespaces
+<a href="https://codespaces.new/geekan/MetaGPT"><img src="https://github.com/codespaces/badge.svg" alt="Open in GitHub Codespaces"></a>
+
+You may use the button above to open this repo in a Codespace
+
+For more info, check out the [GitHub documentation](https://docs.github.com/en/free-pro-team@latest/github/developing-online-with-codespaces/creating-a-codespace#creating-a-codespace).
+
```
Comment (voidking) [inline .devcontainer/README.md:13]: Add init and config steps here.
From the begging to `python startup.py "write a cli snake game"`.

File: README.md [hunk 3]
```diff
@@ -19,6 +19,11 @@
 <a href="https://twitter.com/DeepWisdom2019"><img src="https://img.shields.io/twitter/follow/MetaGPT?style=social" alt="Twitter Follow"></a>
 </p>
 
+<p align="center">
+   <a href="https://vscode.dev/redirect?url=vscode://ms-vscode-remote.remote-containers/cloneInVolume?url=https://github.com/geekan/MetaGPT"><img src="https://img.shields.io/static/v1?label=Dev%20Containers&message=Open&color=blue&logo=visualstudiocode" alt="Open in Dev Containers"></a>
+   <a href="https://codespaces.new/geekan/MetaGPT"><img src="https://github.com/codespaces/badge.svg" alt="Open in GitHub Codespaces"></a>
```
Comment (voidking) [inline README.md:24]: Preview the file. The `Open in GitHub Codespaces` is too large comparing to other buttons.

File: .devcontainer/README.md (hunk 2 above)
Comment (voidking) [reply .devcontainer/README.md:13]: Or tell the user to read the Traditional Install section of the README.

**Comment by voidking** [comment]
LGTM

**Comment by stellaHSR** [comment]
LGTM

**Pull request merged**